configs/altogether.yaml

```diff
@@ -0,0 +1,53 @@
+trainer:
+  precision: bf16-mixed
```

Comment:
did you ever compare bf16-mixed vs bf16-true to see memory overhead change son latter?

Reply:
Yeah. Neither were a visible improvement but I tend to opt towards showing more rather than less features since discoverability is always so-so here.